### PR TITLE
Add pre-build step for nodejs stuff

### PR DESCRIPTION
**Pull request by Morlinest**
As title. Based (=contains) on #2580 (need to be merged first, then I'll rebase this one). Only `.drone.yml` file contains related changes.

Instead of installing node on each PR, use image with nodejs and run nodejs stuff there.

@tboerger Please take a look.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -9,23 +9,29 @@ clone:
     tags: true
 
 pipeline:
+  pre-build:
+    image: webhippie/nodejs:edge
```
Comment (tboerger) [inline .drone.yml]: `edge` doesn't exist, just use `latest` which is based on regular alpine edge packaging.

File: .drone.yml [hunk 2]
```diff
@@ -9,21 +9,27 @@ clone:
...
     commands:
-      - apk -U add nodejs nodejs-npm
-      - npm install
       - make clean
+      - make minify
       - make generate
       - make vet
       - make lint
       - make fmt-check
-      - make stylesheets-check
```
Comment (daviian) [inline .drone.yml:26]: Why removing `stylesheets-check`?

File: .drone.yml [hunk 3]
```diff
@@ -9,21 +9,27 @@ clone:
...
     image: webhippie/golang:edge
     pull: true
     environment:
       TAGS: bindata sqlite
       GOPATH: /srv/app
     commands:
-      - apk -U add nodejs nodejs-npm
-      - npm install
       - make clean
+      - make minify
```
Comment (daviian) [inline .drone.yml]: `minify` can be moved after `stylesheets-check` to prevent unnecessary build step execution

**Review by daviian — CHANGES_REQUESTED** [review]

**Comment by lafriks** [comment]
I'm against that unminified file is stored in repository. Idea is to have all files in that condition so that you can `go get` and `go build` and you don't need any other tools to run it. Of course development requires more tools/steps but just compiling from source should not.

**Comment by tboerger** [comment]
IMHO @lafriks is right.

File: .drone.yml (hunk 2 above)
Comment (Morlinest) [reply .drone.yml:26]: Moved to `pre-build` step.

File: .drone.yml (hunk 3 above)
Comment (Morlinest) [reply .drone.yml]: It has to be runned before `make generate`.

**Comment by Morlinest** [comment]
@lafriks What other tools? The file is there, only it is not minified. Anyway, this should be discussed in #2580.

**Comment by lafriks** [comment]
LGTM

**Review by daviian — APPROVED** [review]

**Comment by daviian** [comment]
LGTM

**Pull request merged**